src/main/java/pl/zz/sharefood/mapper/impl/UserMapperImpl.java

```diff
@@ -0,0 +1,29 @@
+package pl.zz.sharefood.mapper.impl;
+
+import org.springframework.stereotype.Component;
+import pl.zz.sharefood.domain.User;
+import pl.zz.sharefood.domain.dto.UserDto;
+import pl.zz.sharefood.mapper.UserMapper;
+
+@Component
+public class UserMapperImpl implements UserMapper {
+
+    @Override
+    public User userDtoToUser(UserDto userDto) {
+        return User.builder()
+                .email(userDto.getEmail())
+                .name(userDto.getName())
+                .id(userDto.getId())
```

Comment:
Nie chcemy żeby id nam się ustawiało z request'a tylko hibernate niech to nałoży